corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -71,6 +71,10 @@ hqDefine('app_manager/js/forms/case_config_ui', function () {
                         app_manager.updateDOM(data.update);
                         self.requires(requires);
                         self.setPropertiesMap(data.propertiesMap);
+
+                        hqImport('analytix/js/kissmetrix').track.event("Saved question as a Case Property", {
+                            questionsSaved: data.propertiesMap.case.length,
```

Comment:
`data.propertiesMap.case` doesn't exist. Did this seem to work when you tested it?

The saving data is in the actions generated above. There's an `update_case` action that has an `update` property that's an object containing the properties being saved.

Reply:
Hi Jenny, jest it did. The data.propertiesMap.case contained the list of the properties to save. I'll look at this once more.

Reply:
@orangejenny I've tested this and this is the object that I'm getting:
![image](https://user-images.githubusercontent.com/18209901/36016090-bec69c8e-0d71-11e8-943d-3010d5881596.png)
I'll test this deeper, maybe it's due to different place in the UI that I'm testing. In case of trouble, I'll contact you via skype or email.